packages/components/src/core/Tabs/style.tsx

```diff
@@ -94,6 +94,11 @@ export const StyledTab = styled(RawTab, {
         color: black;
       }
 
+      &.Mui-focusVisible {
+        outline: 5px auto Highlight;
+        outline: 5px auto -webkit-focus-ring-color;
```

Comment:
This is currently the same focus outline that buttons have:
<img width="443" alt="Screenshot 2023-11-29 at 12 56 57 PM" src="https://github.com/chanzuckerberg/sci-components/assets/2503289/f007524c-af18-42d1-aa6c-4b7ae15d9278">

I am not a designer, and I'm happy to change this to something else. Figured it made sense for now to keep it simple and match buttons and text inputs.

Note that mui has default focus styles which we're suppressing as part of the theme: https://github.com/chanzuckerberg/sci-components/blob/9451847d508ead93707839e488da799c629914ef/packages/components/src/core/styles/common/defaultTheme.ts#L283-L298

We could turn those back on, although it looks weird and would take tweaking colors for both buttons and tabs.

Reply:
🤔 hmm... Are we re-doing work that we can just get for free from Mui?

Reply:
We definitely are, although I think it would take design and eng work to get back to something that looks reasonable with the defaults for all components.